_i18n/de/resources/moneropedia/clearnet.md

```diff
@@ -1,32 +1,33 @@
 ---
-tags: ["kovri"]
+tags: ["kovri", "Kovri"]
 terms: ["Clearnet"]
-summary: "The Internet in which anonymous overlay networks are built upon"
+summary: "Das Internet, auf welchem anonyme Overlay-Netzwerke aufbauen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-When you use the Internet for things like news, email, social media, and even Monero, you are most likely using a clearnet connection. This means that *all* of your connections can be tracked, traced, and monitored by:
+Wenn du das Internet für Dinge wie das Abrufen der Nachrichten, E-Mails, soziale Medien oder auch Monero nutzt, nutzt du sehr wahrscheinlich eine Clearnet-Verbindung. Dies bedeutet, dass deine *gesamten* Verbindungen aufgespürt, verfolgt und überwacht werden können, und das von:
 
-- your [ISP](https://en.wikipedia.org/wiki/ISP)
-- the website/service/person you're communicating with
-- possibly a [Five Eyes](https://en.wikipedia.org/wiki/5_Eyes) capable entity
+- deinem [Internetprovider](https://de.wikipedia.org/wiki/Internetdienstanbieter)
+- der Webseite/der Person/dem Service, mit der/dem du in Verbindung stehst
+- eventuell einer mit den Möglichkeiten der [Five Eyes](https://en.wikipedia.org/wiki/5_Eyes) befähigten Instanz
 
-and even if you use [HTTPS](https://en.wikipedia.org/wiki/HTTPS) or similar (which *encrypts* your transmission), your route is not hidden nor is it anonymous, thus; it is in the *clear*.
+Selbst wenn du [HTTPS](https://de.wikipedia.org/wiki/Hypertext_Transfer_Protocol_Secure) oder Ähnliches (das deine Datenübertragung *verschlüsselt*) nutzt, ist dein Verlauf weder verborgen noch anonym; es ist also sichtbar und demnach im *"Clear"*.
```

Comment:
```suggestion
Selbst wenn du [HTTPS](https://de.wikipedia.org/wiki/Hypertext_Transfer_Protocol_Secure) oder Ähnliches (das deine Datenübertragung *verschlüsselt*) nutzt, ist deine Verbindung weder verborgen noch anonym; sie ist also sichtbar und demnach im *"Clearnet"*.
```

"... im *Clear*" is weird so I changed it to "... im *Clearnet*" which deviates from the English version but is somewhat less weird.

Reply:
Because it sounded weird to me as well I added the short part ".. ist also sichtbar und demnach ...". I was more thinking about translating it to "im Klaren" because this paragraph seems to not point directly at the Clearnet but more at the fact that your route ist visible, if that makes sense. In the end I left it as "Clear" because while my addition ("sichtbar") explains what's happening here, keeping "Clear" afterwards as it is seemed to be a understandable reference to the "clarity" of the Clearnet - again, if that makes any sense at all.

Reply:
```suggestion
Selbst wenn du [HTTPS](https://de.wikipedia.org/wiki/Hypertext_Transfer_Protocol_Secure) oder Ähnliches nutzt, sind die übertragenen Inhalte zwar verschlüsselt (vor dem "Abhören" durch Dritte geschützt), jedoch ist die Verbindung weder verborgen, noch anonym (Dritte können feststellen, dass die Verbindung existiert und zwischen wem). Daher werden auch HTTPs und ähnliche Technologien dem Clearnet zugeordnet.
```

Another try with some slight deviations from the English version.